airbyte-config/config-models/src/main/resources/types/ActorCatalogFetchEvent.yaml

```diff
@@ -25,3 +25,5 @@ properties:
     type: string
   connectorVersion:
     type: string
+  createdAt:
+    type: integer
```

Comment:
Can we use the OffsetDateTime type here? jsonschema2pojo allows to use a real java type. (see `Extensions` in https://www.jsonschema2pojo.org/). It will match the jooq type.

Reply:
oh I did it this way because I saw other models with datetimes as integers so I thought that was the way we expect datetimes to appear. but i can change it if offset datetimes are preferable